### PR TITLE
updated logging chart readme

### PR DESCRIPTION
**Pull request by paynejacob**
The app readme should have a consistent format across charts.  I used [monitoring](https://raw.githubusercontent.com/rancher/charts/main/charts/rancher-monitoring/app-README.md) as a guide.

### CODE REVIEW

File: packages/rancher-logging/overlay/app-readme.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
-Rancher Logging, powered by the [Banzai Logging Operator](https://banzaicloud.com/docs/one-eye/logging-operator/), allows users to configure complex logging pipelines with a few simple resources.
-To collect logs from a single namespace, users can create an [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) to define where logs are stored and a corresponding [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) to describe what logs to include and what outputs to send to.
-To collect logs from the entire cluster, users can similarly create a [ClusterOutput](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) and a corresponding [ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/). 
+# Rancher Logging
+
+Rancher Logging, powered by the [Banzai Logging Operator](https://banzaicloud.com/docs/one-eye/logging-operator/). It allows users to configure complex logging pipelines with a few simple resources:
```
Comment (nickgerace) [inline packages/rancher-logging/overlay/app-readme.md:3]: nit: ```logging-operator/), allows users to```

File: packages/rancher-logging/overlay/app-readme.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+
+- [Output](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/output_types/) configures where to store the collected logs for the namespace it is created in
```
Comment (nickgerace) [inline packages/rancher-logging/overlay/app-readme.md:5]: nit: ```where to store the collected logs within the namespace they are created in.```

File: packages/rancher-logging/overlay/app-readme.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+- [Flow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/flow_types/) configures what logs in the namespace are sent to the configured outputs. 
+- [ClusterOutput](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusteroutput_types/) configures where to store logs collected for the cluster.
+- [ClusterFlow](https://banzaicloud.com/docs/one-eye/logging-operator/crds/v1beta1/clusterflow_types/) configures what logs to direct to the150150 cluster outputs.
```
Comment (nickgerace) [inline packages/rancher-logging/overlay/app-readme.md:8]: Is this intentional? ```the150150```

**Review by nickgerace — DISMISSED** [review]

**Review by rmweir — COMMENTED** [review]
Wondering about the `the150150` thing as well, otherwise will LGTM.

File: packages/rancher-logging/overlay/app-readme.md (hunk 3 above)
Comment (paynejacob) [reply packages/rancher-logging/overlay/app-readme.md:8]: That is a puppy walking across the keyboard I'll remove it.

**Review by nickgerace — APPROVED** [review]

**Comment by deniseschannon** [comment]
Closing in favor of https://github.com/rancher/charts/pull/827/files

**Pull request closed without merging**